src/util.mjs

```diff
@@ -0,0 +1,26 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import { Buffer } from 'node:buffer'
+import { randomFillSync } from 'node:crypto'
+
+export const randomId = (function () {
+  const buffer = Buffer.alloc(10)
+  return function () {
+    return randomFillSync(buffer).toString('hex')
```

Comment:
Let's move bach to Math.random, as we don't need crypto here.

Reply:
Ok.